src/tests/Loader/classloader/MethodImpl/InternalMethodImplTest.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+ <PropertyGroup>
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
+    <CLRTestTargetUnsupported Condition="'$(RuntimeFlavor)' != 'coreclr'">true</CLRTestTargetUnsupported>
```

Comment:
@jkoritzinsky @trylek What is the recommended way to enable test on CoreCLR only?

Reply:
I'd remove the RequiresProcessIsolation property and a SkipOnMono attribute on the [Fact] method.

Reply:
Tried removing RequiresProcessIsolation, but got 'Unhandled exception. System.MissingMethodException: Entry point not found in assembly 'InternalMethodImplTest, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.' when running the test

Reply:
Hmm, how did you run the test? Without process isolation, the test should become part of the merged test group ```src/tests/Loader/Loader.csproj``` so this would be the right project to build and run unless you're using the environment variable ```BuildAsStandalone=true``` which basically suppresses all test merging and builds each test to be separately runnable.

Reply:
We need to disable the test on native AOT as well. What's the best way to do that?

Reply:
> We need to disable the test on native AOT as well. What's the best way to do that?

We don't seem to have a dedicated Skip attribute for NativeAOT, so perhaps with an explicit condition:

```c#
if (TestLibrary.Utilities.IsNativeAot)
{
    return 100; // unsupported on NativeAOT
}
```